spec/scenarios/codeblock/fenced/list-continuation-yaml.adoc

```diff
@@ -0,0 +1,37 @@
+. Identify the outdated VMIs:
++
+[,bash]
+----
+  $ kubectl get vmi -l kubevirt.io/outdatedLauncherImage --all-namespaces
```

Comment:
If code blocks are supposed to be indented in Markdown, why are the code block lines not indented, while the bash line is indented?

Reply:
It's because the Markdown is still invalid. kramdown-asciidoc can only convert valid Markdown.